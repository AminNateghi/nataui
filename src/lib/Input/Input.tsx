export const Input = () => {
  return (
    <>
      <p>input component</p>
    </>
  );
};
