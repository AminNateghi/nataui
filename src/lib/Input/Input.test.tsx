import renderer from 'react-test-renderer'
import { describe, expect, test } from 'vitest'
import { Input } from './Input'

describe('Input', () => {
  test('Input component renders correctly', () => {
    const component = renderer.create(
      <Input />
    )

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})