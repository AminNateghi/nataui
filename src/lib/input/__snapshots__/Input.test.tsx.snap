// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Input > Input component renders correctly 1`] = `
<p>
  input component
</p>
`;
